### DISCUSSION TITLE
I am new to coding and open source, where should I start?

### DISCUSSION THREAD

**Discussion by YuriDevAT** [YurisCodingClub/accessibility-mentor#11] [General]
1. If you are completely new to coding, meaning, you just started to learn how to write HTML and CSS, maybe a bit of JavaScript, you cab get familiar with Open Source by contributing to the HTML Tag Quiz project.
Link to repository https://github.com/YurisCodingClub/html-tag-quiz

2. For information on how to clone a repository, please follow the steps on the repository's README
Link to README https://github.com/YurisCodingClub/html-tag-quiz/blob/main/README.md

3. For general information on how to contribute to a project in YurisCodingClub, please read the contribution guideline thoroughly.
Link to the contribution guide https://github.com/YurisCodingClub/accessibility-mentor/blob/main/CONTRIBUTING.md

4. I don't understand the terms used in an issue. What should I do? If you are a complete beginner in coding or open source, don't worry.
-   Let's start with the following: Look up what semantic HTML means.
-   Think about what a navigation actually is (a list of links). Use the corresponding semantic HTML for        it.
-   You can find everything needed to fulfill this issue by looking the terms up in [w3schools](https://www.w3schools.com/) or [MDN](https://developer.mozilla.org/en-US/).
-   If these sites are new to you, don't worry. You will get used to them over time.

Happy coding 🙂

**Comment by BamML** [comment]
Im confused on how it is supposed to work... could you please try to explain?

**Reply by YuriDevAT** [reply]
Hi @BamML. With what exactly are you confused? How to start with basic website creation? Or how to clone a project and work on it?

For how to get started in coding (HTML, CSS, and JS) I would suggest working through the docs of w3schools. This is also where I started back then https://www.w3schools.com/html/default.asp

When you want to know how to start contributing to a project you are interested in, follow the steps described on the READ.ME file of the projects.
Let's say you want to contribute to my HTML Quiz Project (https://github.com/YurisCodingClub/html-tag-quiz), you have to fork the repo, download it and work on it on you own local computer. https://github.com/YurisCodingClub/html-tag-quiz/blob/main/README.md#how-to-contribute

Hope this helps 🙂
